php/PaypalIPN.php

```diff
@@ -75,31 +75,31 @@ public function verifyIPN()
         $raw_post_data = file_get_contents('php://input');
         $raw_post_array = explode('&', $raw_post_data);
         $myPost = array();
+        $magic_quotes_enabled = false;
```

Comment:
Magic quotes `has been DEPRECATED as of PHP 5.3.0 and REMOVED as of PHP 5.4.0.`
I don't see the value in adding a check here - I would rather specify a minimum PHP version, 
http://php.net/manual/en/security.magicquotes.php